altos-rust/port/cortex-m0/src/peripheral/usart/control.rs

```diff
@@ -250,43 +524,83 @@ impl CR2 {
 /// CR3
 // ------------------------------------
 
-#[derive(Copy, Clone)]
+/// Defines the possible HardwareFlowControl configurations for the Usart.
+/*  None: No hardware flow control
```

Comment:
I just realized you previously had these as doc comments and then you moved them up here... :1st_place_medal: